render/detailed/links.go

```diff
@@ -160,7 +161,19 @@ func metricQuery(summary NodeSummary, n report.Node, metricID string) string {
 		return ""
 	}
 
-	return strings.Replace(queries[metricID], "{{label}}", summary.Label, -1)
+	namespace, _ := n.Latest.Lookup(kubernetes.Namespace)
+	name, _ := n.Latest.Lookup(docker.ContainerName)
+	replacements := map[string]string{
+		"{{label}}":         summary.Label,
+		"{{namespace}}":     namespace,
+		"{{containerName}}": name,
+	}
+
+	q := queries[metricID]
+	for old, new := range replacements {
```

Comment:
What about using a `Replacer` for this? https://golang.org/pkg/strings/#Replacer